packages/eas-cli/src/commands/build/submit.ts

```diff
@@ -75,13 +75,31 @@ export default class BuildSubmit extends Command {
     'apple-id': flags.string({
       description: 'Your Apple ID username (you can also set EXPO_APPLE_ID env variable)',
     }),
-    'apple-app-specific-password': flags.string({
```

Comment:
Normally when there is login passed via params and password via env we are adding info about env to the description of the login param and throw an exception if param was passed and env was not.

In this case we don't have pair of values like that but maybe we could do that based on `app-apple-id`, I'm not sure if it makes sense? Are those two values related in a meaningful way?

Reply:
App specific password isn't related to any of these flags directly. It's just required either as  env var or prompted during runtime.